Dockerfile

```diff
@@ -350,10 +414,10 @@ RUN --mount=type=cache,sharing=shared,id=cargo_registry_index,target=${CARGO_HOM
         --package drive-abci \
         ${FEATURES_FLAG} \
         --locked && \
-    cp /platform/target/${OUT_DIRECTORY}/drive-abci /artifacts/ && \
-    # Remove target directory to save space
-    rm -rf target && \
-    if [[ -x /usr/bin/sccache ]]; then sccache --show-stats; fi
+    cp target/${OUT_DIRECTORY}/drive-abci /artifacts/ && \
+    if [[ -x /usr/bin/sccache ]]; then sccache --show-stats; fi && \
+    # Remove /platform to reduce layer size
+    rm -rf /platform
```

Comment:
I think this might not really decrease layer size,  as these files are already inside previous layer, and layers are, in fact, diffs. I suspect this can even increase the layer size a bit, as we need to store info about removed files.

Reply:
I tested, no difference between `rm platform` and `rm target`, leaving as it is